app/experimenter/experiments/tests/api/v3/test_serializers.py

```diff
@@ -239,6 +363,7 @@ def test_serializer_creates_experiment_and_sets_slug_and_changelog(self):
             "features": ["picture_in_picture", "pinned_tabs"],
             "firefox_min_version": FIREFOX_VERSION,
             "firefox_channel": Experiment.CHANNEL_RELEASE,
+            "variants": FAKE_VARIANTS,
```

Comment:
@k88hudson @tiftran Don't we need to add checks that the variants were created correctly in this test?